src/commands/admin/generate-api-keys/readme.ts

```diff
@@ -1,19 +1,87 @@
-export default `#### Usage
+import { buildExamples } from '../../../lib/docgen/buildExamples';
+import { ScopeName, TRANSCEND_SCOPES } from '@transcend-io/privacy-types';
+import type { GenerateApiKeysCommandFlags } from './impl';
 
-\`\`\`sh
-transcend admin generate-api-keys --email=test@transcend.io --password=$TRANSCEND_PASSWORD \\
-   --scopes="View Email Templates,View Data Map" --apiKeyTitle="CLI Usage Cross Instance Sync" -file=./working/auth.json
-\`\`\`
+const examples = buildExamples<GenerateApiKeysCommandFlags>(
+  ['admin', 'generate-api-keys'],
+  [
+    {
+      description: 'Generate API keys for cross-instance usage',
+      flags: {
+        email: 'test@transcend.io',
+        password: '$TRANSCEND_PASSWORD',
+        scopes: `${TRANSCEND_SCOPES[ScopeName.ViewEmailTemplates].title},${
+          TRANSCEND_SCOPES[ScopeName.ViewDataMap].title
+        }`,
+        apiKeyTitle: 'CLI Usage Cross Instance Sync',
+        file: './working/auth.json',
+      },
+    },
```

Comment:
this is the typesafe example, which gets docgen'd into the \`\`\`sh block. It will generate line-break the args if it is >117 chars on a single line (and thus would require horizontal scrolling on github)